frontend/components/Layout/AppBar/Buttons/UserButton.vue

```diff
@@ -13,6 +13,11 @@
       </app-bar-button-layout>
     </template>
     <v-list dense>
+      <v-list-item>
+        <v-list-item-title class="text-subtitle-1 text-center">
+          {{ auth.users[auth.currentUserIndex].Name }}
```

Comment:
Why not use the getter we have in the store for the current user?

Reply:
Because that was a quick draft and I didn't notice it haha. It's fixed